Source/Core/Core/CoreTiming.cpp

```diff
@@ -479,8 +479,8 @@ void Idle()
 
 	idledCycles += DowncountToCycles(PowerPC::ppcState.downcount);
 	PowerPC::ppcState.downcount = 0;
-
-	Advance();
+	if(_M_ARM_32) //Temp Fix
```

Comment:
@lioncash : Without these two lines ARMv7 recompiler won't work. I cant remove the check but i don't want create a regression on other platformer although the impact will be zero. By the way i don't know if Arm64 recompiler work with Magumagu fix, there is only HdkR which use Arm64 recompiler.